roles/keystone-defaults/defaults/main.yml

```diff
@@ -83,6 +83,7 @@ keystone:
     sensu_checks:
       check_keystone_api:
         criticality: 'critical'
+    cron_bin_path:  "/sbin:/usr/sbin:{{ (os == 'rhel' ) | ternary('/usr/bin/', '/usr/local/bin/') }}"
```

Comment:
This var name is confusing a bit, is the purpose of this to be able to run the keystone-manage command? If so, /usr/bin and /usr/local/bin should be in $PATH on both ubuntu and rhel.